### PR TITLE
AV-194094 : Cover profile generation for AKO UT's from Makefile for coverage report in Sonarqube

### PR DESCRIPTION
**Pull request by DixitAakash**
This PR is for generating cover profile for AKO UT's from Makefile. An additional -coverprofile flag has been added to each go test command. This will generate coverage report (cover.out file) for each go test command. These coverage reports will be consumed by a downstream Jenkins job and fed to Sonarqube for generating aggregated code coverage.

### CODE REVIEW

**Comment by DixitAakash** [comment]
build ako

File: Makefile [hunk 1]
```diff
@@ -175,63 +175,63 @@ k8stest:
 	sudo docker run \
 	-w=/go/src/$(PACKAGE_PATH_AKO) \
 	-v $(PWD):/go/src/$(PACKAGE_PATH_AKO) $(BUILD_GO_IMG) \
-	$(GOTEST) -v -mod=vendor $(PACKAGE_PATH_AKO)/tests/k8stest -failfast
+	$(GOTEST) -v -mod=vendor $(PACKAGE_PATH_AKO)/tests/k8stest -failfast -coverprofile cover-1.out -coverpkg=./...
```
Comment (arihantg) [inline Makefile:178]: should we add cover-*.out in .gitignore?

Comment (DixitAakash) [reply Makefile:178]: Done

**Comment by DixitAakash** [comment]
build ako

**Review by arihantg — APPROVED** [review]

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**